tests/Console/Command/FixCommandTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Console\Command;
+
+use PhpCsFixer\ConfigurationException\InvalidConfigurationException;
+use PhpCsFixer\Console\Application;
+use PhpCsFixer\Console\Command\FixCommand;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @internal
+ *
+ * @covers \PhpCsFixer\Console\Command\FixCommand
+ */
+final class FixCommandTest extends TestCase
+{
+    /**
+     * @var Application
+     */
+    private $application;
+
+    protected function setUp()
+    {
+        $this->application = new Application();
+    }
+
+    public function testEmptyRulesValue()
+    {
+        $this->doTestExecute(
+            array('--rules' => ''),
+            array(
+                'class' => InvalidConfigurationException::class,
+                'regex' => '#^Empty rules value is not allowed\.$#',
```

Comment:
not sure about having same utests for 2 layers - currently, it's for command and resolver. but the logic is the same, and command is passing values to resolver.

Reply:
Thought about it, but I went for this route.
When passing "php-cs-fixer --rule=&nbsp;&nbsp;&nbsp;" than the value of `--rule=` will be set to "" (and not "&nbsp;&nbsp;&nbsp;").
This behavior seems to be bound to SF/console and I thought it best to cover it in a test as well.
So in theory the application will never passes "&nbsp;&nbsp;&nbsp;" as value for `rules` to the configuration resolver, however people may use the configuration outside of the application so may pass it.